### DISCUSSION TITLE
Equivalent to VDataIterator?

### DISCUSSION THREAD

**Discussion by lbm-services** [quasarframework/quasar#16979] [General - Components / Directives / etc] [answered]
I need to upgrade an older Vuetify 2 app and am considering migrating to Quasar.
After playing around for some days now, I like Quasar and its robust layout design. Some design features seem to be easier to achieve compared to Vuetify. Implementing socket.io was a bit of a hassle though, but is finally working.

What I miss now is an equivalent to VDataIterator, which works like VDataTable , meaning it comes with pagination, filtering and search capabilities out of the box, but is not restricted to table/tr/td html. 
Is there a workaround for this in Quasar?

**Comment by yusufkandemir** [comment, marked as answer]
I was in the same spot about 4 years and 9 months ago, and the solution is the same to this day, which works great. See [QTable Grid style with slots](https://quasar.dev/vue-components/table#example--grid-style-with-slot). When using `grid` prop, you can use `#item` slot to use any content you would like(_usually q-card, but not limited to that_), displayed in any dimensions you want(_through the flex system_).

**Comment by lbm-services** [comment]
Great. Since its basically not a table anymore, I would have exported this feature to a new component with a different name.
